docs/events-actions.md

```diff
@@ -0,0 +1,368 @@
+# Events and Actions
+
+There are many events on GitHub which gets subscribed to webhooks. One of these events is triggered, GitHub sends a HTTP POST payload to the webhook's configured URL.
+
+While configuring a webhook, one can choose which events you would like to receive payloads for. Only subscribing to the specific events you plan on handling is useful for limiting the number of HTTP requests to your server. You can even opt-in to all current and future events. You can change the list of subscribed events through the API or UI anytime. 
+
+Also, each event corresponds to a certain set of actions that can happen to your organization and/or repository.
+
+Given below is a list of GitHub webhook events and actions related to the events.
+  * [`*`](https://github.com/aps120797/probot/blob/master/docs/events-actions.md#-wildcard-event)
```

Comment:
I do think these links need to be changed to work on the probot website, as opposed to link to your fork of probot. Some find and replace should do the trick! The url should be something like `https://probot.github.io/docs/events-actions#event-name`

Reply:
Good catch 👍

Reply:
Oh my! Nice catch. Going to fix it. Thanks! 😄

Reply:
@aps120797 I think you can just use relative links. Something like 

```
  - [`*`](#-wildcard-event)
  - [`commit_comment`](#commit_comment)
```

Also, it seems like the other docs use `-` rather than `*` for unordered lists. See [docs/deployment.md](https://github.com/probot/probot/blob/master/docs/deployment.md) for an example.

Reply:
@philschatz , fixed both. :smile: